### DISCUSSION TITLE
Swap number and function layers to the right hand

### DISCUSSION THREAD

**Discussion by cjohnsonuk** [manna-harbour/miryoku#292] [Q&A] [answered]
I've just finished building a Kyria my first split programable keyboard. I have chosen to go with a Miryoku based layout.  I have all 1u keys so have a few spares for additional shortcuts etc to minimise the use of layers (eg dedicated copy and paste (because I do a lot of that)).  

1) I was thinking of moving the space key to the right hand thumb cluster on one of the extra keys as I've noticed that on a standard qwerty non split keyboard I use my right hand thumb  (my dominant one) for space.  

2) I've also thought about moving  mouse and button layers to the left hand and move numbers and symbols that I'll use more to my dominant right hand.  

3) I'm also thinking that as I'm used to Fn+BKSP for DEL on my apple keyboard I might set that up in the thumb cluster too so the left hand thumb home key layer has DEL instead of backspace on the right hand thumb cluster.  

I'm thinking these changes would ease the training and switching between kyria and built in laptop keyboards as I'll not need to retrain my thumb when stitching between laptop and Kyria keyboards. 

Before I go and do this does anyone think there will be any issues or  know if this will break the principles behind the Miryoku  original layout??

**Comment by manna-harbour** [comment, marked as answer]
There is an option to swap all layers and Base layer thumbs between hands: https://github.com/manna-harbour/miryoku/tree/master/docs/reference#flip. That's the recommended option as it won't introduce any issues.

You can also substitute custom layers and map unused keys. See https://github.com/manna-harbour/miryoku/discussions/85.

Also see:
- https://github.com/manna-harbour/miryoku/discussions/133
- https://github.com/manna-harbour/miryoku/discussions/159
- https://github.com/manna-harbour/miryoku/discussions/189
- https://github.com/manna-harbour/miryoku/discussions/269
- https://github.com/manna-harbour/miryoku/discussions/288
